test/test_sorting_topic.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from src.sorting_topic import selection_sort
+
+
+class SortingTopicTest(unittest.TestCase):
```

Comment:
Given you're cramming everything into a single file here, this test class is going to get chonky. Consider pulling out the list of cases into a class constant and using them all with `subTest`. Every sorting algorithm you add is going to be another ~30 lines of the same tests, otherwise.